Kateryna_Volynets/Friends_and_Enemies.py

```diff
@@ -0,0 +1,9 @@
+names_friends = ["John", "Marta", "James"]
+names_enamies = ["John", "Johnatan", "Artur"]
+for i in names_friends:

```

Comment:
do not use general naming. a better practice is to use more understandable names. In our case it is name variable instead of i